src/transformers/quantizers/quantizer_quanto.py

```diff
@@ -193,7 +213,7 @@ def _process_model_after_weight_loading(self, model):
 
     @property
     def is_trainable(self, model: Optional["PreTrainedModel"] = None):
-        return False
+        return True
```

Comment:
switching to optimum quanto makes them trainable? 👁️ 👁️ 
If yes, should be version specific no? Or am I missing something?!

Reply:
Trainable with peft ! No changes in quanto, just in peft cc @BenjaminBossan